content/source/docs/enterprise/admin/agents-on-tfe.html.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: "enterprise"
+page_title: "Terraform Cloud Agents on TFE - Infrastructure Administration - Terraform Enterprise"
+---
+
+# Terraform Cloud Agents on TFE
+
+Terraform Cloud Agents allow Terraform Enterprise to communicate with isolated,
+private, or on-premises infrastructure. By deploying lightweight agents within a
+specific network segment, you can establish a simple connection between your
+environment and Terraform Enterprise which allows for provisioning operations and
+management.
+
+Terraform Cloud Agents on TFE behave identically to [Terraform Cloud Agents on TFC](https://www.terraform.io/docs/cloud/agents/index.html). Please reference this
+document for getting started steps and operational requirements. Exceptions and
+additional requirements are listed below.
```

Comment:
```suggestion
Terraform Cloud Agents on TFE behave very similarly to [Terraform Cloud Agents on TFC](https://www.terraform.io/docs/cloud/agents/index.html), with a few exceptions listed below. This page contains the steps to get started and operational requirements for Terraform Enterprise. 
```

Reply:
I tried to get too tricky with phrasing this. I felt like putting "This page contains the steps to get started and operational requirements for Terraform Enterprise" made the "this" sound like the _current_ page, not the linked page. Ah well.